test/integration/addon_configuration_test.go

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) 2022 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package integration
+
+import (
+	"encoding/json"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/stolostron/governance-policy-framework/test/common"
+)
+
+var _ = Describe("GRC: [P1][Sev1][policy-grc] Test add-on configuration", Serial, Label("SVT"), func() {
```

Comment:
I realized that our integration tests don't run in parallel anyway, so `Serial` isn't strictly necessary here... but I think it is very helpful to denote that this test uses "shared" resources and would impact any other tests if they were running.

Reply:
I agree--it'd be really nice if they were parallel eventually, so putting `Serial` here is a good forward-looking thing to do.